{{cookiecutter.project_slug}}/docs-jupyter-book/_toc.yml

```diff
@@ -0,0 +1,8 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: intro
+chapters:
+# Add below the .md and .ipynb pages of your Jupyter-book.
+- file: 
```

Comment:
not sure .. but maybe it will raise an error as there is no file value here. have you tested it locally?
if it is not working, no worries, you can work on that in a follow-up pr.

also, as you can see here in github, it shows a red circle here that means that it doesn't have a empty end line here. 

but for now just add this empty end line here please